lib/bolt/plugin/module.rb

```diff
@@ -158,20 +158,20 @@ def process_params(task, opts)
         # opts are passed directly from inventory but all of the _ options are
         # handled previously. That may not always be the case so filter them
         # out now.
-        _meta, params = opts.partition { |key, _val| key.start_with?('_') }.map(&:to_h)
-        metaparams = {}
+        meta, params = opts.partition { |key, _val| key.start_with?('_') }.map(&:to_h)
+        validate_params(task, params)
+
         # Send config with `_config` when config is defined in bolt_plugin.json
         # Otherwise, merge config with params
         # TODO: remove @send_config when deprecated
         if @send_config
-          metaparams['_config'] = config if config?
+          params['_config'] = config if config?
```

Comment:
I think that `metaparams` used to collect all of the `_*` params, but now we're doing that with the `partition` command. What this is doing is adding `_config` and `_boltdir` to the task params, which I think is part of what's causing the travis failures.